api/nakadi-event-bus-api.yaml

```diff
@@ -0,0 +1,486 @@
+swagger: '2.0'
+info:
+  title: Nakadi Event Bus API Definition
+  description: |
+    -------------------------------
+    Nakadi Event Bus API Definition
+    -------------------------------
+
+    * Low-level API
+    * Stantardised event format (see #/definitions/Event)
+    * Partition strategy: event store will infer the partition from the `partitioning_key` in the message body
+
+  version: '0.4'
+  contact:
+    name: Team Aruha @ Zalando
+    email: nakadi-users@zalando.de
+schemes:
+  - https
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  oauth2:
+    type: oauth2
+    flow: implicit
+    authorizationUrl: 'https://auth.example.com/oauth2/tokeninfo'
```

Comment:
We explicitly removed the scopes here, as I wanted to avoid discussions on that topic in the first stage. 

We could also go an easy way and start with two scopes here, that generally allows reading from the system or publishing to the system.

We could also a scope for "adding topics"